shared_model/backend/protobuf/from_old.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <map>
+#include <set>
+#include <string>
+#include "backend/protobuf/permissions.hpp"
+
+namespace shared_model {
+  namespace interface {
+    namespace permissions {
+      static inline Role fromOldR(const std::string &s) {
```

Comment:
I suggest not to use such shortening - this one and `fromOldG` just below are quite confusing imo.

Reply:
they're temporary functions and should be removed after